src/contrib/cluster/Akka.DistributedData/GCounter.cs

```diff
@@ -47,8 +47,6 @@ public sealed class GCounter :
         FastMerge<GCounter>,
         IRemovedNodePruning<GCounter>,
         IEquatable<GCounter>,
-        IComparable<GCounter>,
```

Comment:
Removed `IComparable<GCounter>` - you can't really compare and order the two since this doesn't take into account the different versions recorded across nodes. Let users extract the underlying `ulong` value if they need that functionality.